packages/project-access/src/project/cap.ts

```diff
@@ -0,0 +1,98 @@
+import { join } from 'path';
+import { FileName } from '../constants';
+import type { CapCustomPaths, CapProjectType, csn, Package } from '../types';
+import { fileExists, readJSON } from '../file';
+import { loadModuleFromProject } from './moduleLoader';
+
+/**
+ * Returns true if the project is a CAP Node.js project.
+ *
+ * @param packageJson - the parsed package.json object
+ * @returns - true if the project is a CAP Node.js project
+ */
+export function isCapNodeJsProject(packageJson: Package): boolean {
+    return !!(packageJson?.cds || packageJson?.dependencies?.['@sap/cds']);
```

Comment:
packageJson is defined based on the function signature, so you can remove the check, or is the signature incorrect?

Reply:
fixed